### ISSUE TITLE
"Pause playback after each video" resetting watchtime / "Stop playback after one video" closing instead of showing suggestions?

### ISSUE THREAD

**Issue by Magmagan** [yuliskov/SmartTubeNext#1335]
The options "Pause playback after each video" and "Pause playback after each non-playlist video" currently move the player time back to the start, effectively not registering that the video was watched to completion on YouTube. This then leads YouTube to suggest already watched videos, and distort recommendations based on watch time.

Was the commit https://github.com/yuliskov/SmartTubeNext/commit/7f7761053bc88b075b07cf599c2bef8ef1935e28 inevitable? Is `.setPositionMs(0);` absolutely necessary?

https://github.com/yuliskov/SmartTubeNext/blob/92db89446f1c09d319d8fec3fd1b812ff22f33b2/common/src/main/java/com/liskovsoft/smartyoutubetv2/common/app/models/playback/managers/VideoLoaderManager.java#L186-L194

Conversely, "Stop playback after one video" does solve the first issue, but then recommendations for continued watching are inaccessible since the video player has been closed.

Perhaps this suggested behavior was intended, but isn't working? Is `isSuggestionsShown()` supposed to stop the player for closing if there are recommendations?

https://github.com/yuliskov/SmartTubeNext/blob/92db89446f1c09d319d8fec3fd1b812ff22f33b2/common/src/main/java/com/liskovsoft/smartyoutubetv2/common/app/models/playback/managers/VideoLoaderManager.java#L176-L185

**Comment by tbolick** [comment]
Assuming I am reading this correctly, I have the same issue.  I have it stopping after one video, and sometimes (I think after 'sleep' or something) instead of going back to the 'home' screen, it just closes.  However, it doesn't fully close because re-opening it also does not go to the 'home screen'.  If I reboot my ChromeCast 2/Google TV (where it is running) It starts back at the same place.  Can we at least get a way to close/kill it from the after-video screen?

**Comment by TarkusLV** [comment]
Yeah, I used to use the Pause feature, but found myself with a ton of videos that YouTube thought I partially watched, instead of completely watched.  Thats not an issue when using Stop instead.  I would prefer to pause at the end, though.  Not just for the related recommendations (which don't really work these days anyway), but also to give the video a Like if I hadn't done it yet.
